### ISSUE TITLE
Invizible not auto starting

### ISSUE THREAD

**Issue by dhammel** [Gedsh/InviZible#125]
Hello,

I have a pixel 6 android 12 January 2022 update. 
Invizible pro version 5.5.0 Stable

I use vpn mode, but also occurs for root mode.

I use dnscrypt 24/7, tor is only occasional so I have the start dnscrypt on boot option enabled.

I have battery optimzation settings set to unrestricted. I have no cleaner app installed that would kill its process. I do have the system set to start invizible each boot.

After reboot, when i open invizible it will give me an error that android system killed it, and network settings have been restored.

Have any idea what may be going?

I will try and post its log here soon, if i can.

Thank you

**Comment by Gedsh** [comment]
Hello,

> After reboot, when i open invizible it will give me an error that android system killed it

You should usually wait a few minutes for Android to start InviZible. So just be patient. If that doesn't help, try turning on Always On VPN in Android settings.

> Have any idea what may be going?

Please send me those logs, I'll check.

**Comment by dhammel** [comment]
Its working, took about 5-6 minutes. Other app like rethinkdns starts in ~ 10 seconds. Seemed like a long time for it to start.

I tried to get logs, but upon pressing collect logs and saving to root of storage, it saved a txt file 22 bytes, all that was in it was letters 'Pk'.

**Comment by Gedsh** [comment]
> Its working, took about 5-6 minutes. Other app like rethinkdns starts in ~ 10 seconds.

This is strange. Usually InviZible is started along with other applications. Have you changed the delay option in Fast Settings?

> I tried to get logs, but upon pressing collect logs and saving to root of storage, it saved a txt file 22 bytes

I can't explain it until I see the log of saving logs. Looks like a loop :)

**Comment by dhammel** [comment]
> Have you changed the delay option in Fast Settings?

The delay is set to 0

Logs:
 https://drive.google.com/file/d/103aaa9Ji0caVubCk2-ZCr5VZ9ZQejJks/view?usp=drivesdk

**Comment by Gedsh** [comment]
> Logs

Yep. Logs are empty. It looks like your device does not even allow using the internal memory. You can only try some external log saving application if it will work.
